MemberExpectations.md

```diff
@@ -0,0 +1,63 @@
+It is important that members of our leadership groups (TSC, CommComm) act
+in a way that not only complies with the
+[Code of Conduct](https://github.com/nodejs/TSC/blob/master/CODE_OF_CONDUCT.md)
+but that supports the vision, mission and operating principles of the Node.js project.
+
+Members are encouraged to have individual opinions and are encouraged
+to express those opinions during discussions within the organization. While
+discourse is encouraged internally, members also have the obligation outside
+of the organization (for example in social media) to be respectful of the
+decisions made within the projects's documented processes.  A member who does
+not support a team decision or current team value may express the member's
+opposition within the team in an appropriate manner, but must be extremely
+careful when expressing the opinion externally.  External communication must
+not be done with the intent of undermining a decision, project values, or
+actions of the team. If another team member expresses that they believe external
+communications appear to have the intent to undermine the committee
+it is the responsibility of the individual expressing their opinions to rectify
+the situation immediately. If a member feels that there is no recourse aside
+from going public regarding an extremely contentious issue then they should
+resign from the team before doing so.
+
+In addition to being willing to act as a team player, members of our
+leadership groups must conduct themselves in a professional and
+respectful manner. Some general
+guidelines include:
+
+- Serve as ambassadors of the vision, mission and operating
+  principles of the Node.js project.
+- Remediate quickly when you realize you made a mistake. Leaders are human,
+  and they will make mistakes - however they should act swiftly to
+  acknowledge mistakes and correct them. Most often this can mitigate
+  any damage done.
+- Aim to remediate first and then discuss.  If other members of the
+  team express concerns about actions, acknowledge their concerns by
+  stopping the actions in question and then discuss within the team
+  to come to a common agreement.
+- Treat all community members with respect, consideration, and highest
```

Comment:
I 100% agree with this line, but it's a highly subjective and abstract statement. My idea of respect, consideration, and ethical conduct may vary from yours. Should this line mention something to this effect?

Reply:
perhaps explicit language that you are expected to correct if others feel behavior falls outside that norm

Reply:
I think we already have this covered in the bullet about about remediate first.